src/wmnds/components/breadcrumb/_properties.njk.json

```diff
@@ -0,0 +1,41 @@
+{
+  "wmndsBreadcrumbProps": [
+    {
+      "name": "items",
+      "type": "array",
+      "description": "Specifies the section which the current page falls under. If this is the same as the page title it will not display.",
```

Comment:
```suggestion
      "description": "<strong>Required.</strong> Specifies the section which the current page falls under. If this is the same as the page title it will not display.",
```

Reply:
Should this use the same text as gov to be more clear?
![image](https://user-images.githubusercontent.com/13015201/102902272-afb28f80-4466-11eb-9722-a93e9c904812.png)